dcompiler.dd

```diff
@@ -851,17 +870,15 @@ $(H2 $(LEGACY_LNAME2 compiling_phobos, compiling-phobos, Compiling Phobos))
 $(WINDOWS
 $(CONSOLE
 cd $(DMDDIR)\src\druntime
-make -f win32.mak DMD=$(DMDDIR)\$(OS)\bin\dmd
+make -f win32.mak
 cd ..\phobos
-make -f win32.mak DMD=$(DMDDIR)\$(OS)\bin\dmd
+make -f win32.mak
 )
 )
 $(UNIX
 $(CONSOLE
-cd ~/$(DMDDIR)/src/druntime
-make -f posix.mak DMD=~/$(DMDDIR)/$(OS)/bin/dmd
 cd ../phobos
-make -f posix.mak DMD=~/$(DMDDIR)/$(OS)/bin/dmd
+make -f posix.mak
```

Comment:
DRuntime is built automatically since a few months (though currently the source can't be built - https://forum.dlang.org/post/bmfzbxcmwuhcduldjtdx@forum.dlang.org)